web/package.json

```diff
@@ -24,23 +23,28 @@
     "gulp-util": "^3.0.4",
     "json-loader": "^0.5.4",
     "lodash": "^4.0.0",
+    "pure-render-decorator": "^1.1.1",
```

Comment:
Seems like a better idea. 15.3.0 breaks react-virtualized though... so probably wait till that gets fixed.

I'm fine with getting rid of pure-render-decorator though. This is copied over from older code.